codeflash/cli_cmds/cmd_init.py

```diff
@@ -587,6 +587,16 @@ def configure_pyproject_toml(setup_info: SetupInfo) -> None:
         )
     elif formatter == "don't use a formatter":
         formatter_cmds.append("disabled")
+    if formatter in ["black", "ruff"]:
+        try:
+            result = subprocess.run([formatter], capture_output=True, check=False)
+            click.echo(f"✅ Formatter exists on system")
+            click.echo()
+        except FileNotFoundError as e:
+            click.echo(f"⚠️ Formatter not found: {formatter}")
```

Comment:
add a helpful message to help the user resolve the problem. Suggest installing it or ensuring it is present